client/packages/system/src/Stock/DetailView/DetailView.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect } from 'react';
+import {
+  TableProvider,
+  createTableStore,
+  useTranslation,
+  DetailTabs,
+  // usePluginElements,
+  // usePluginEvents,
+  // PluginEventListener,
+  useBreadcrumbs,
+  useParams,
+  useConfirmationModal,
+  useNotification,
+} from '@openmsupply-client/common';
+import { ActivityLogList } from '@openmsupply-client/system';
+import { AppBarButtons } from './AppBarButtons';
+import { useStockLine } from '../api';
+import { StockLineForm } from '../Components/StockLineForm';
+import { LedgerForm } from '../Components/Ledger';
+
+export const StockLineDetailView: React.FC = () => {
+  const { id } = useParams();
+  const {
+    query: { data, isLoading },
+    draft,
+    resetDraft,
+    updatePatch,
+    isDirty,
+    update: { update, isUpdating },
+  } = useStockLine(id);
+  // const { dispatchEvent, addEventListener, removeEventListener } =
+  //   usePluginEvents();
+  // const [hasChanged, setHasChanged] = useState(false);
```

Comment:
Was this a previous edition of `isDirty`?

Reply:
Yeah I guess it was. Sorry, I actually assumed it was something to do with the plugins, but at closer inspection it's not -- gone now 💨